### PR TITLE
Fix context

### PR DESCRIPTION
**Pull request by varadeth**
@prasanna-egov @saiprakash-v

### CODE REVIEW

File: backend/vc-management-service/src/configs/config.js [hunk 1]
```diff
@@ -13,14 +13,17 @@ const JWT_PUBLIC_KEY = process.env.AUTH_PUBLIC_KEY;
 const BASE_URL = process.env.VC_MANAGEMENT_SERVICE_BASE_URL || '/vc-management/';
 const REQUEST_TIMEOUT = process.env.REQUEST_TIMEOUT || 10000;
 const TENANT_NAME = process.env.TENANT_NAME || 'Tenant';
-
+const ROOT_URL = process.env.ROOT_URL || `http://vc-management-service:${PORT}`;
+const IS_MINIO = process.env.IS_MINIO === 'true';
```
Comment (prasanna-egov) [inline backend/vc-management-service/src/configs/config.js:17]: True is boolean not a string.

File: backend/vc-management-service/src/routes/context.routes.js [hunk 2]
```diff
@@ -11,17 +11,28 @@ const redisService = require('../services/redis.service');
...
-        accessKey: config.MINIO_ACCESSKEY,
-        secretKey: config.MINIO_SECRETKEY,
-    });
-    if(!(await minioClient.bucketExists(constants.MINIO_BUCKET_NAME)))
-        await minioClient.makeBucket(constants.MINIO_BUCKET_NAME);
-    if(config.REDIS_ENABLED) {
-        await redisService.initRedis({REDIS_URL: config.REDIS_URL});
+    try {
+        let minioOptions = {
+            endPoint: config.MINIO_URL,
```
Comment (prasanna-egov) [inline backend/vc-management-service/src/routes/context.routes.js:16]: so is this working with PORT ?

Comment (prasanna-egov) [reply backend/vc-management-service/src/routes/context.routes.js:16]: In SRC it seems to be working just with the HTTP URL hence checking

**Review by prasanna-egov — APPROVED** [review]

**Review by saiprakash-v — APPROVED** [review]

File: backend/vc-management-service/src/routes/context.routes.js (hunk 2 above)
Comment (varadeth) [reply backend/vc-management-service/src/routes/context.routes.js:16]: no. it gave error invalid endpoint. below I have added a flat of IS_MINIO and then only added port

**Pull request merged**